src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { BrowserRouter, Switch, Route } from "react-router-dom";
+import './App.css';
+import Index from './pages';
+
+/*Create a basic route for the home page */
+function App() {
+  return (
+    <BrowserRouter>
```

Comment:
You're not actually using the browser router for routing here, its just unused code. React ROuter is used to turn the "single page" web app into a "multi-page" web app without having to create multiple html files.